networking/advertise-service-ips.md

```diff
@@ -49,10 +50,23 @@ If your {{site.prodname}} deployment is configured to peer with BGP routers outs
 
 ### Before you begin...
 
-- [Configure BGP peering]({{ site.baseurl }}/networking/bgp) between {{site.prodname}} and your network infrastructure
+**Required**
+- [Configure BGP peering]({{site.baseurl}}/networking/bgp) between {{site.prodname}} and your network infrastructure
 - For ECMP load balancing to services, the upstream routers must be configured to use BGP multipath.
 - You need at least one external node outside the cluster that acts as a router, route reflector, or ToR that is peered with calico nodes inside the cluster.
-- Services must be configured with the correct service type (“Cluster” or “Local”) for your implementation. For `externalTrafficPolicy: Local`, the service must be type `LoadBalancer` or `NodePort`.
+- Services must be configured with the correct service mode (“Cluster” or “Local”) for your implementation. For `externalTrafficPolicy: Local`, the service must be type `LoadBalancer` or `NodePort`.
```

Comment:
"service mode" -> "traffic policy" (to align with the API terminology)

Reply:
Fixed.

Reply:
looks like this didn't get pushed?